backend/src/main/kotlin/fr/gouv/cnsp/monitorfish/domain/entities/logbook/messages/PNO.kt

```diff
@@ -11,6 +11,9 @@ import java.time.ZonedDateTime
 
 // TODO Rename to `LogbookMessageValueForPno`.
 class PNO() : LogbookMessageValue {
+    var authorizedPortEntrance: Boolean? = null
```

Comment:
Juste je chipote pour le naming en `is` ou `has`, par ex:

- `hasPortEntranceAuthorization`
- `hasPortLandingAuthorization`